src/components/Images.vue

```diff
@@ -298,30 +308,65 @@ export default {
     deleteImage(obj) {
       this.currentCommand = `delete ${ obj.imageName }:${ obj.tag }`;
       this.mainWindowScroll = this.$refs.fullWindow.parentElement.parentElement.scrollTop;
+      this.postOpSuccessHandler = this.postDeleteSuccessHandler;
       this.startRunningCommand('delete');
       ipcRenderer.send('confirm-do-image-deletion', obj.imageName.trim(), obj.imageID.trim());
     },
+    postDeleteSuccessHandler() {
+      if (this.imageManagerOutput === '') {
+        this.closeOutputWindow(null);
+      }
+    },
     doPush(obj) {
       this.currentCommand = `push ${ obj.imageName }:${ obj.tag }`;
+      this.mainWindowScroll = this.$refs.fullWindow.parentElement.parentElement.scrollTop;
       this.startRunningCommand('push');
       ipcRenderer.send('do-image-push', obj.imageName.trim(), obj.imageID.trim(), obj.tag.trim());
     },
     doBuildAnImage() {
       this.currentCommand = `build ${ this.imageToBuild }`;
       this.fieldToClear = 'imageToBuild';
+      this.postCloseOutputWindowHandler = this.scrollToPulledImage(this.imageToBuild);
       this.startRunningCommand('build');
       ipcRenderer.send('do-image-build', this.imageToBuild.trim());
     },
     doPullAnImage() {
       this.currentCommand = `pull ${ this.imageToPull }`;
       this.fieldToClear = 'imageToPull';
+      this.postCloseOutputWindowHandler = this.scrollToPulledImage(this.imageToPull);
       this.startRunningCommand('pull');
       ipcRenderer.send('do-image-pull', this.imageToPull.trim());
     },
+    scrollToPulledImage(imageToPull) {
```

Comment:
This should be named `scrollToImage(imageName)`, since this is also used for built images.

Reply:
That will change -- there's going to be a `scrollToImage` method, but this will get a different name.